test/distributed/_tensor/test_xla_integration.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
```

Comment:
Tested locally with CloudTPU, and this should be skipped in the non-xla CI workflows.